src/libgrate/matrix.c

```diff
@@ -143,7 +143,7 @@ void mat4_perspective(struct mat4 *m, float fov, float aspect,
 	m->xx = cotangent / aspect;
 	m->yy = cotangent;
 	m->zz = -(far + near) / depth;
-	m->zw = -1.0f;
-	m->wz = -2.0f * near * far / depth;
+	m->zw = -2.0f * near * far / depth;
+	m->wz = -1.0f;
```

Comment:
Yeah. I'm fine with making things consistently row-major, though. I'd like if we could be consistent with OpenGL also, but it's less important ;)

So, let's do this as-is, and possible change our layout later, OK?